vaadin-side-nav-flow-parent/vaadin-side-nav-testbench/src/main/java/com/vaadin/flow/component/sidenav/testbench/SideNavItemElement.java

```diff
@@ -53,17 +56,28 @@ public boolean isActive() {
         return hasAttribute("active");
     }
 
-    public String getLabel() {
-        final WebElement unnamedSlot = getWrappedElement().getShadowRoot()
-                .findElement(By.cssSelector("slot:not([name])"));
-        return (String) executeScript(
-                "return arguments[0].assignedNodes()[0].textContent;",
-                unnamedSlot);
+    @Override
+    public void click() {
+        click(1, 1);
+    }
+
+    public void clickAnchor() {
+        executeScript("arguments[0].click();", getWrappedElement()
+                .getShadowRoot().findElement((By.cssSelector("a"))));
     }
```

Comment:
If `click` does the same as `navigate` did before, why do we need this?

The click could probably be triggered through the Selenium API rather than running a script.

Reply:
During the Start integration, we discovered that `navigate` will not work with `click(1,1)` implementation when the drawer is hidden. However, the `click` does not trigger navigation at all, if not overridden. I don't have a quick solution to do them both as I haven't debugged it to see why it is the case. But in the unofficial `AppNav` component in Start had its separate TestBench classes and it used the logic of `getAnchor().click()` to navigate. The `click` not working by itself can be addressed separately I guess. Also, it might be a bug in the component itself.

So, I can 
- revert the name to `navigate`
- use the logic in `clickAnchor` to solve the hidden drawer issue 
- remove the `click` override
- use only `navigate` on tests

How does it sound?